src/main/java/nl/knaw/dans/dvcli/DdDataverseCli.java

```diff
@@ -73,7 +79,8 @@ public void configureCommandLine(CommandLine commandLine, DdDataverseCliConfig c
                 .addSubcommand(new CollectionView()))
             .addSubcommand(new CommandLine(new DatasetCmd(dataverseClient))
                 .addSubcommand(new DeleteDraft())
-            );
+            )
+            .addSubcommand(new CommandLine(new NotificationTruncate(database)));
```

Comment:
start.sh truncate-notifications --help

works, but the command does not show up on 

    start.sh --help

Reply:
Must have mixed up things, it is ok now.